lib/developer_portal/config/initializers/liquid.rb

```diff
@@ -25,7 +25,6 @@
     Liquid::Tags::AuthorizeNetForm,
     Liquid::Tags::PaymentExpressForm,
     Liquid::Tags::BraintreeCustomerForm,
-    Liquid::Tags::OgoneForm,
```

Comment:
@akostadinov I think this is not right

we previously had issues with removing that Liquid, so we brought it back.

Reply:
@mayorova that's right

Reply:
didn't we live for 1 month without it?

Reply:
@akostadinov nope, we removed it, then realized that it breaks some existing portals, so we rolled it back. the Authorize.net removals did NOT touch the OgoneForm liquid.

Reply:
@akostadinov this is how it looks currently https://github.com/3scale/porta/blob/a31cd4011bba74b7bbdb152db549980f1fd5f35c/lib/developer_portal/config/initializers/liquid.rb